src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -15,15 +16,15 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
i am not sure that eager is a good idea, but i don't know
let Sofia resolve my question